data/json/furniture_and_terrain/terrain-windows.json

```diff
@@ -1105,11 +1095,9 @@
       "TRANSPARENT",
       "NOITEM",
       "CONNECT_TO_WALL",
-      "THIN_OBSTACLE",
       "BARRICADABLE_WINDOW_CURTAINS",
       "BLOCK_WIND",
       "REDUCE_SCENT",
-      "PERMEABLE",
```

Comment:
This window would be permeable because there is no glass, just bars and a curtain.